projects/go-tester/src/styles.scss

```diff
@@ -2,6 +2,8 @@
 @import url('https://fonts.googleapis.com/css?family=Roboto:300,400,700');
 @import url('https://fonts.googleapis.com/icon?family=Material+Icons');
 
+@import "~@tangoe/gosheets/gosheets";
```

Comment:
Nice! This should allow us to refactor out some additional code.

Reply:
Yessssssir!